openhcl/underhill_attestation/src/igvm_attest/mod.rs

```diff
@@ -35,12 +33,36 @@ pub enum Error {
     },
 }
 
+/// Rust-style enum for `IgvmAttestReportType`
+pub enum ReportType {
```

Comment:
I don't think this needs to be pub?

Reply:
But why do those fields need to be pub? you init it using a constructor... so unless there's some code outside this module that's reaching into the helper directly, it seems like some (or all) of these fields can be left private?

Reply:
We only need to expose `runtime_claims_hash`.  Do you suggest having a get function for this field and make all the fields private?

Reply:
Yeah, having all fields private + a getter method seems like the right call